apps/src/gamelab/GameLabP5.js

```diff
@@ -451,13 +452,40 @@ GameLabP5.prototype.registerP5EventHandler = function (eventName, handler) {
   this.p5[eventName] = handler;
 };
 
+GameLabP5.prototype.setP5FrameRate = function () {
+  if (!this.p5) {
+    return;
+  }
+  if (this.stepSpeed < 1) {
+    // TODO: properly handle overriding frameRate
+    this.prevFrameRate = this.p5.frameRate();
+    this.p5.frameRate(1);
+  } else {
+    this.p5.frameRate(this.prevFrameRate || 30);
```

Comment:
Where were we setting 30 as the default framerate before?  Should this 30 be deduplicated with that one?

Reply:
👍